crates/router/src/services/email/types.rs

```diff
@@ -237,7 +245,7 @@ pub struct VerifyEmail {
 /// Currently only HTML is supported
 #[async_trait::async_trait]
 impl EmailData for VerifyEmail {
-    async fn get_email_data(&self) -> CustomResult<EmailContents, EmailError> {
+    async fn get_email_data(&self, base_url: &str) -> CustomResult<EmailContents, EmailError> {
```

Comment:
Can we take base URL in the `VerifyEmail` type instead?

Reply:
We can take in types. Can be refactored in separate PR.